subo/command/create_runnable.go

```diff
@@ -45,6 +45,20 @@ var langAliases = map[string]string{
 	"gr":         "grain",
 }
 
+// Error for build command CreateRunnableCmd failures
+type CreateRunnableError struct {
+	Path string // The ouput directory for build command CreateRunnableCmd.
+	Err  error  // The original error.
+}
+
+// Error acts as a cleanup function for CreateRunnableError
+func (err CreateRunnableError) Error() string {
+	if cleanup_err := os.RemoveAll(err.Path); cleanup_err != nil {
+		fmt.Println("🚫 failed to clean up runnable outputs")
+	}
+	return err.Err.Error()
+}
```

Comment:
I'm not sure about having side effects in an `Error()` method. Maybe something like this instead?

We also shouldn't be printing here. We can just return the error.

```go
func NewCreateRunnableError(path string, err error) CreateRunnableError {
	if cleanupErr := os.RemoveAll(path); cleanupErr != nil {
		err = errors.Wrap(err, "failed to clean up runnable outputs")
	}
	return CreateRunnableError{Path: path, error: err}
}
```